pkg/gofr/migration/manager_test.go

```diff
@@ -0,0 +1,29 @@
+package migration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"gofr.dev/pkg/gofr/container"
+	"gofr.dev/pkg/gofr/testutil"
+)
+
+func Test_getMigratorDatastoreNotInitialised(t *testing.T) {
+	logs := testutil.StdoutOutputForFunc(func() {
+		container, _ := container.NewMockContainer(t)
```

Comment:
please name it to `mockContainer` as name **container** is colliding with imported package name